crypto/src/conversation.rs

```diff
@@ -80,6 +80,12 @@ pub struct MlsConversationCreationMessage {
     pub message: MlsMessageOut,
 }
 
+#[derive(Debug)]
+pub struct MlsConversationLeaveMessage {
```

Comment:
You could reorder fields so that they have the same order as the sister struct in ffi lib